src/stp/plays/referee_specific/FreeKickThem.cpp

```diff
@@ -2,31 +2,76 @@
-        std::make_unique<role::Defender>("defender_2"), std::make_unique<role::Defender>("blocker_0"),  std::make_unique<role::Defender>("blocker_1"),
-        std::make_unique<role::Defender>("blocker_2"),  std::make_unique<role::Defender>("blocker_3"),  std::make_unique<role::Defender>("blocker_4"),
-        std::make_unique<role::Defender>("blocker_5"),  std::make_unique<role::Formation>("offender")};
+        std::make_unique<role::Keeper>("keeper"),
+        std::make_unique<role::Defender>("defender_0"),
+        std::make_unique<role::Defender>("defender_1"),
+        std::make_unique<role::Defender>("defender_2"),
+        std::make_unique<role::Defender>("blocker_0"),
+        std::make_unique<role::Defender>("blocker_1"),
+        std::make_unique<role::Defender>("blocker_2"),
+        std::make_unique<role::Defender>("blocker_3"),
+        std::make_unique<role::Defender>("blocker_4"),
+        std::make_unique<role::Defender>("blocker_5"),
+        std::make_unique<role::Formation>("offender")
+    };
+
+    initRoles();  // DONT TOUCH.
 }
 
-uint8_t FreeKickThem::score(world::World *world) noexcept { return 100; }
+Dealer::FlagMap FreeKickThem::decideRoleFlags() const noexcept {
+    Dealer::FlagMap flagMap;  // DONT TOUCH.
+
+    /// Flags that have a factor and a weight linked to it, can be given to a role
+    Dealer::DealerFlag closeToOurGoalFlag(DealerFlagTitle::CLOSE_TO_OUR_GOAL, DealerFlagPriority::HIGH_PRIORITY);
+    Dealer::DealerFlag closeToBallFlag(DealerFlagTitle::CLOSE_TO_BALL, DealerFlagPriority::HIGH_PRIORITY);
+
+    /// Creation flagMap. Linking roles to role-priority and the above created flags, can also force ID {roleName, {priority, flags, forceID}}
+    flagMap.insert({"keeper", {DealerFlagPriority::KEEPER, {}}});
+    flagMap.insert({"defender_0", {DealerFlagPriority::HIGH_PRIORITY, {closeToOurGoalFlag}}});
+    flagMap.insert({"defender_1", {DealerFlagPriority::HIGH_PRIORITY, {closeToOurGoalFlag}}});
+    flagMap.insert({"defender_2", {DealerFlagPriority::HIGH_PRIORITY, {closeToOurGoalFlag}}});
+    flagMap.insert({"blocker_0", {DealerFlagPriority::HIGH_PRIORITY, {closeToBallFlag}}});
+    flagMap.insert({"blocker_1", {DealerFlagPriority::HIGH_PRIORITY, {closeToBallFlag}}});
+    flagMap.insert({"blocker_2", {DealerFlagPriority::HIGH_PRIORITY, {closeToBallFlag}}});
+    flagMap.insert({"blocker_3", {DealerFlagPriority::HIGH_PRIORITY, {closeToBallFlag}}});
+    flagMap.insert({"blocker_4", {DealerFlagPriority::HIGH_PRIORITY, {closeToBallFlag}}});
+    flagMap.insert({"blocker_5", {DealerFlagPriority::HIGH_PRIORITY, {}}});
+    flagMap.insert({"offender", {DealerFlagPriority::HIGH_PRIORITY, {closeToBallFlag}}});
+
+    return flagMap;  // DONT TOUCH.
+}
+
+uint8_t FreeKickThem::score(PlayEvaluator& playEvaluator) noexcept {
+    calculateInfoForScoredRoles(playEvaluator.getWorld());  /// DISABLE IF NOT USED
```

Comment:
This function is not used and as an empty function in your header, so you can disable/comment it out